src/WebJobs.Script/Rpc/LanguageWorkerProcess.cs

```diff
@@ -79,10 +80,12 @@ public void StartProcess()
 
                 // Register process only after it starts
                 _processRegistry?.Register(_process);
+                return Task.CompletedTask;
```

Comment:
Why did you make this async? No async operations are performed?

Reply:
Made this aysnc as starting process can run in a background thread